style.css

```diff
@@ -124,81 +190,174 @@ hr {
+        height: calc(100% - (var(--icon-padding) * 2));
+    }
 }
-.about-li-icon img {
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    right: 0;
-    left: 0;
-    margin: auto;
-    object-fit: scale-down;
-    object-position: center;
-    width: calc(100% - (var(--icon-padding) * 2));
-    height: calc(100% - (var(--icon-padding) * 2));
-}
-#experience ul {
-    padding-left: 0;
+
+#experience {
+    ul {
+        padding-left: 0;
+    }
 }
-#education {
-    margin-bottom: 4em;
+
+#education,
+#experience {
+    margin-bottom: var(--spacer);
 }
-#education ul {
-    padding-left: 0;
+
+#education {
+    ul {
+        padding-left: 0;
+    }
 }
 
 /* Projects page */
-.projects-li {
-    margin-top: 2em;
+#projects-content {
+    h2 {
+        margin-top: calc(var(--spacer) / 2);
+    }
 }
-.projects-project-title {
-    vertical-align: bottom;
-    display: inline;
+
+.projects-li {
+    margin-top: calc(var(--spacer) / 2);
 }
+
 .project-header {
     display: flex;
     align-items: flex-end;
+    gap: 1em;
+    justify-content: space-between;
     border-bottom: 1px solid var(--color-fg);
-    margin-bottom: 2em;
+    margin-bottom: calc(var(--spacer) / 4);
     padding-bottom: 0.2em;
+
+    >a {
+        text-decoration: none;
+        flex-basis: 33%;
+
+        &:visited {
+            color: var(--color-link);
+        }
+
+        &:hover,
+        &:focus-visible {
+            color: var(--color-link-hover);
+        }
+
+        h3 {
+            font-size: var(--font-size-med);
+            margin: 0;
+
+            &::before {
+                content: '> '
+            }
+        }
+    }
+
+    .project-header-ul {
+        text-align: right;
+        display: flex;
+        flex-wrap: wrap;
+        gap: .3em;
+        justify-content: flex-end;
+        flex-basis: 66%;
+        flex-grow: 1;
+        text-indent: 0;
+        padding: 0;
+
+        li {
+            display: flex;
+            margin: 0;
+            flex-direction: column;
+            align-items: center;
+            padding: .1em .5em;
+        }
+    }
 }
-.project-header a {
-    text-decoration: none;
-}
-.project-header-ul {
-    text-align: right;
-    flex-grow: 1;
-}
+
 .lang {
     font-weight: bold;
-    font-size: 12pt;
+    font-size: var(--font-size-sm);
     color: white;
     display: inline-block;
     border-radius: 0.3em;
     padding: 0.2em 0.6em;
     margin: 0.1em 0;
 }
+
 .lang-rust {
     background-color: var(--color-rust);
 }
+
 .lang-c {
     background-color: var(--color-c);
 }
+
 .lang-cpp {
     background-color: var(--color-cpp);
 }
+
 .lang-shell {
     background-color: var(--color-shell);
 }
+
 .lang-python {
     background-color: var(--color-python);
 }
+
 .lang-java {
     background-color: var(--color-java);
 }
+
 .lang-go {
     background-color: var(--color-go);
 }
+
 .lang-gdscript {
     background-color: var(--color-gdscript);
 }
+
+/* Large-Breakpoint */
+@media(min-width: 767px) {
```

Comment:
this is where I then updated things to make the fonts change on the large breakpoint